python-scripts/src/main/python/dataproducts/resources/queries/content_list.py

```diff
@@ -14,8 +14,8 @@ def init():
                 },
                 {
                     "type": "in",
-                    "dimension": "contentType",
-                    "values": ["Resource", "ExplanationResource", "FocusSpot", "PracticeQuestionSet", "eTextBook", "LearningOutcomeDefinition", "PracticeResource", "ExperientialResource", "SelfAssess", "CuriosityQuestionSet"]
+                    "dimension": "mimeType",
+                    "values": "mimetype_filter"
```

Comment:
Rename the current query to content_list_v1.py and create a new query

Reply:
Should I still be reading the whitelist filter from the config object or have it in the new query itself?